doc/requirements_and_expectations.md

```diff
@@ -0,0 +1,30 @@
+# Requirements for SQL Query Builder
+
+## 1. Functional Requirements
+
+- Provide an interface for selecting tables from the database.
+- Enable creating and configuring relationships (`JOIN`) between selected tables.
+- Support column selection, adding filters (`WHERE`), setting sorting (`ORDER BY`), and grouping (`GROUP BY`) parameters.
+- Automatically generate SQL queries based on user actions.
+- Execute SQL queries and display the results.
+- Automatically detect relationships between tables using foreign keys.
+- Display error messages for incorrect query configurations.
```

Comment:
Only this messages?
also why there may be error messages for incorrect query configuration?
And next question is what does "query configuration" mean here?